swagger/index.js

```diff
@@ -131,13 +133,21 @@ module.exports = yeoman.Base.extend({
           name: model.name,
           plural: model.plural,
           base: model.base || 'PersistedModel',
-          facetName: 'server', // hard-coded for now
+          facetName: 'common', // hard-coded for now
```

Comment:
To be honest, I am still not convinced, but don't want to block the progress here.

One last question - have you considered users upgrading their projects from the old generator to this new version? In my understanding, moving the generated code from `server` to `common` means that users will have to rewrite most of their operation implementation and move it from `server` to `common` too. OTOH, maybe that's not a concern because the layout of the generated code is already too different from the previous code-gen version?